contentcuration/contentcuration/frontend/channelEdit/views/template/MoveModal.vue

```diff
@@ -0,0 +1,213 @@
+            <VIcon>assignment</VIcon>
+            <VBtn flat color="primary" class="button">{{ $tr("moveClipboard") }}</VBtn>
+            <v-spacer />
+            <VBtn flat @click="$emit('cancelMove')">{{ $tr("cancel") }}</VBtn>
+            <VBtn color="primary" class="white--text" @click="moveNodes">{{ $tr("moveHere") }}</VBtn>
+          </BottomToolBar>
+        </v-card>
+      </VDialog>
+    </v-layout>
+  </v-container>
+</template>
+
+<script>
+import { mapGetters, mapActions, mapState } from 'vuex';
+import { RouterNames } from '../../constants';
+import BottomToolBar from '../../../shared/views/BottomToolBar';
+import NewTopicModal from './NewTopicModal';
+
+export default {
+  name: 'MoveModal',
+  components: {
+    NewTopicModal,
+    BottomToolBar,
+  },
+  props: {
+    nodeId: {
+      type: String,
+      required: true,
+    },
+    showDialog: {
+      type: Boolean,
+    },
+  },
+  data() {
+    return {
+      showNewTopicModal: false,
+    };
+  },
+  computed: {
+    ...mapGetters('contentNode', ['getContentNode', 'getContentNodeChildren', 'getTreeNode']),
+    currentNode() {
+      return this.getContentNode(this.nodeId);
+    },
+    children() {
+      return this.getContentNodeChildren(this.nodeId);
+    },
+    crumbs() {
+      const trail = [];
+      var node = this.getContentNode(this.nodeId);
+      var inTree = this.getTreeNode(node.id);
+      trail.unshift({
+        text: node.title,
+        disabled: false,
+        to: this.nextItem(node),
+      });
+      while (inTree !== undefined) {
+        node = this.getContentNode(inTree.parent);
+        inTree = this.getTreeNode(node.id);
+        trail.unshift({
+          text: node.title,
+          disabled: false,
+          to: this.nextItem(node),
+        });
+      }
+      return trail;
+    },
+  },
+  created() {
+    if (!this.currentNode) {
+      this.loadContentNode(this.nodeId).then(this.getChildren);
+      var node = this.getContentNode(this.nodeId);
+      while (node.parent !== null) {
+        this.loadContentNode(node.parent).then(() => {
+          node = this.getContentNode(node.parent);
+        });
+      }
+    } else {
+      this.getChildren();
+    }
+  },
+  methods: {
+    ...mapActions('contentNode', ['createContentNode', 'loadContentNode', 'loadChildren']),
+    nextItem(child) {
+      return {
+        name: RouterNames.SANDBOX,
+        params: {
+          nodeId: child.id,
+        },
+      };
+    },
+    getChildren() {
+      if (this.currentNode && this.currentNode.has_children) {
+        return this.loadChildren({
+          parent: this.nodeId,
+          channel_id: this.$store.state.currentChannel.currentChannelId,
+        });
+      }
+      return Promise.resolve();
+    },
+    showTopicModal() {
+      this.showNewTopicModal = true;
+    },
+    cancelTopic() {
+      this.showNewTopicModal = false;
+    },
+    createTopic(title) {
+      console.log('create title!');
```

Comment:
This isn't going to fully work yet (updates coming in #1734), but you could change this to `this.createContentNode({ parent: this.nodeId, kind: 'topic', title }).then(() => {})` to avoid linting errors around console.log statements and the unused `title` variable. Could you also add a TODO comment for us to come back to later?